nebula-logger/main/logger/classes/LogEntryBuilder.cls

```diff
@@ -171,25 +171,25 @@ public without sharing virtual class LogEntryBuilder {
         this.logEntryEvent.LimitsSoqlQueryRowsMax__c         = Limits.getLimitQueryRows();
         this.logEntryEvent.LimitsSoslSearchesMax__c          = Limits.getLimitSoslQueries();
         this.logEntryEvent.Locale__c                         = UserInfo.getLocale();
-        this.logEntryEvent.LoginDomain__c                    = sessionMap == null ? null : sessionMap.get('LoginDomain');
-        this.logEntryEvent.LoginHistoryId__c                 = sessionMap == null ? null : sessionMap.get('LoginHistoryId');
-        this.logEntryEvent.LoginType__c                      = sessionMap == null ? null : sessionMap.get('LoginType');
-        this.logEntryEvent.LogoutUrl__c                      = sessionMap == null ? null : sessionMap.get('LogoutUrl');
+        this.logEntryEvent.LoginDomain__c                    = sessionMap?.get('LoginDomain');
```

Comment:
I just realized you're using the new safe navigation now for null checks - love it!

Reply:
it's a huge win for readability!